index.html

```diff
@@ -6,12 +6,193 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <script src="https://kit.fontawesome.com/39fd49c23e.js" crossorigin="anonymous"></script>
+
 </head>
 
 <body>
-    <nav class="navbar">
-        <h2>hypnosis</h2>
-    </nav>
+    <div class="nav-content">
+        <nav class="navbar">
+            <button><i class="fa-sharp fa-solid fa-magnifying-glass"></i></button>
+            <h2>hypnosis</h2>
+            <button><i class="fa-sharp fa-solid fa-bars"></i></button>
+        </nav>
+    </div>
+    <fieldset class="content">
+        <header>
+            <h1>Stand up for Somalia</h1>
+            <hr>
+            <img src="christine.png">
+            <h2>Christine Kolesnikov</h2>
+            <p>Hypnosis - Senior Editor</p>
+            <button><a href=""><i class="fa-brands fa-facebook"></i> Shere</a></button>
+            <button><a href=""><i class="fa-brands fa-twitter"></i> Tweet</a></button>
+            <button><a href=""><i class="fa-brands fa-linkedin"></i> Share</a></button>
+            <button><a href=""><i class="fa-sharp fa-solid fa-link"></i> Copy</a></button>
```

Comment:
Hi @Albinpirela , great job so far, please if you have any list of components in your project you need to put a list.
```
<ul>
   .
   .
   .
   .
</ul>
```

Reply:
gracias por la sugerencia @hendridg voy a hacer esa corrección de inmediato 📑

Reply:
thanks for the suggestion @hendridg I'll make that correction right away 📑